bundles/org.openhab.binding.mqtt.espmilighthub/src/main/java/org/openhab/binding/mqtt/espmilighthub/internal/ConfigOptions.java

```diff
@@ -21,6 +23,7 @@
  */
 @NonNullByDefault
 public class ConfigOptions {
+    public BigDecimal duvThreshold = BigDecimal.ONE;
     public int whiteThreshold = -1;
     public int whiteSat = 32;
     public int whiteHue = 35;
```

Comment:
Please adjust the default value for these 2 variables to -1  as this is the new default you defined in the XML file.